third-party/unittest/googlemock/README.LLVM

```diff
@@ -10,14 +10,10 @@ Cleaned up as follows:
 # Remove all the unnecessary files and directories
 $ rm -f CMakeLists.txt configure* Makefile* CHANGES CONTRIBUTORS README README.md .gitignore
 $ rm -rf build-aux make msvc scripts test docs
-$ rm -f `find . -name \*\.pump`
 $ rm -f src/gmock_main.cc
 
 # Put the license in the consistent place for LLVM.
 $ mv LICENSE LICENSE.TXT
 
 Modified as follows:
 * Support for std::begin/std::end in gmock-matchers.h
```

Comment:
Is this still true? If it is, it sounds like the kind of patch that ought to be made in upstream googletest. If not, please remove this line.

Reply:
Yes, gmock-matchers.h still have this part patched, see https://gist.github.com/zeroomega/fb24b1d1c4252b852200e15fda29384d#file-gtest_llvm_modification-patch-L251 . When locally test it, without the patch, LLVM unit tests will have build errors.

Reply:
That seems odd, and worth following up with upstream googletest. In the meantime, I think it would be clearer to future maintainers if we did something to clearly identify the code blocks that were changed, and why. For example,
`// LLVM local change to support std::begin/std::end`
followed by the new code, and then the old code commented-out. That way diffs will provide clues, and merges/commits will have conflicts that someone will need to sort out properly.

Reply:
I put the local changes into commented blocks in change in amend commit: 2fe51a37bad9fc6aa08f09b4b6f137034e2c8140 in this PR, please take a look.

Reply:
I looked at that specific commit, seems fine.